index.js

```diff
@@ -0,0 +1,108 @@
+/*1-Write a console.log statement saying "Hello World!" for each language that you know.*/
+console.log('Halo, dunia!'); //Indoniasia
+console.log('Ciao, mondo!'); //Italian
+console.log('Hola, mundo!'); //Spanish
+
+//2-onsider the following code://
+console.log("I'm awesome");
+//Declare a variable age and initialize it with an integer, using these exact steps://
+var age;
+console.log('The Value of Age will be: undefined');
+console.log(age);
+age= 28;
+console.log('The Value of Age will be: 28');
+console.log(age);
+//4-Declare a variable name and assign a string to it.//
+var name = 'Hamam Alsamel';
+console.log('The Name will be: Hamam Alsamel');
+console.log(name);
+var x= name;
+console.log('The Value of X will be: Hamam Alsamel');
+console.log(x);
+//5-How do you round the number 7.25, to the nearest integer (i.e., whole number)?
+var number= 7.25;
+console.log(number);
+var z= 7;
+console.log(z);
+if (number>=z){
+  console.log(number);
+  var newvalue = number;
+  console.log(newvalue);
+}
+//6- Arrays//
+
+const list = ['','',''];
+console.log('The array has no value becauswe its empty');
+console.log(list);
+
+const favouritanimal = ['Cat','Dog','Birds','elephant'];
+console.log(favouritanimal);
+favouritanimal.push('Kitten')
+console.log(favouritanimal);
+
+//7-More strings: Let's consider the following string: let myString = "this is a test".
+let mystring= "This is a test";
+console.log(mystring);
+console.log(mystring.length);
+console.log('the length of my string');
+
+//8.If x equals 7, and the only other statement is x = x % 3, what would be the new value of x?
+
+var x=7;
+x = x%7;
```

Comment:
I'm a little confused about what you're testing in this question! note that this should be `x = x % 3`, not `x = x % 7`